docs/framework/interop/com-callable-wrapper.md

```diff
@@ -84,6 +84,8 @@ End Class
 // Implicitly extends System.Object.
 public class Mammal
 {
+    public Mammal() {}
```

Comment:
If a class doesn't define any other constructor, it will have a default public parameterless constructor. So I don't think adding the constructor here and also below is necessary.

Reply:
You are right, this default public parameterless constructor has been added only as a visual example for perception. It can be deleted.